client/my-sites/checkout/composite-checkout/components/item-variation-picker/types.ts

```diff
@@ -6,6 +6,10 @@ export type WPCOMProductVariant = {
 	price: number;
 	pricePerMonth: number;
 	pricePerYear: number;
+	regularPrice: number;
```

Comment:
I think we should be more clear about what "regular" means here, either by providing a more clear name and/or adding a comment. There's so many versions of a price we could have. I'm guessing this is in comparison to the introductory offer price? So maybe `priceAfterIntroductoryOffer`?